daceml/transformation/constant_folding.py

```diff
@@ -140,6 +140,8 @@ def apply(self, sdfg: dace.SDFG):
             sub_state = sub_sdfg.add_state()
 
             node_copy = copy.deepcopy(node)
+            # always use ORT for constant folding
+            node_copy.implementation = "onnxruntime"
```

Comment:
not a big fan if it requires us to have ort installed for any daceml operation

Reply:
Unfortunately the dace implementations are still not stable enough to work for all cases (especially in edge cases with scalars, which is where constant folding is usually applied)